src/Js__JSON.res

```diff
@@ -1,13 +1,26 @@
 type t = Js.Json.t
 
+type jsonReviver
+external asJsonReviver: 'a => jsonReviver = "%identity"
+type jsonReplacer
+external asJsonReplacer: 'a => jsonReplacer = "%identity"
+
 @val external parseExn: string => t = "JSON.parse"
+@val external parseExnWithReviver: (string, jsonReviver) => t = "JSON.parse"
 @val external stringify: t => string = "JSON.stringify"
 @val external stringifyWithIndent: (t, @as(json`null`) _, int) => string = "JSON.stringify"
+@val external stringifyWithReplacer: (t, jsonReplacer) => string = "JSON.stringify"
+@val external stringifyWithReplacerAndIndent: (t, jsonReplacer, int) => string = "JSON.stringify"
```

Comment:
Maybe it's better to have one method instead of 4:
```rescript
@val external stringify: (t, ~jsonReplacer: jsonReplacer=?, ~indent: int=?, ()) => string = "JSON.stringify"
```
@MoOx @bloodyowl what do you think?

Reply:
I'd like to avoid having an extra unit argument at the end for the general use-case, it'd make the API a feel bit weird for beginners.

Reply:
Makes sense, also it will generate bigger js output